clap-v3-utils/src/input_parsers/signer.rs

```diff
@@ -286,8 +319,16 @@ pub fn try_pubkeys_sigs_of(
     matches: &ArgMatches,
     name: &str,
 ) -> Result<Option<Vec<(Pubkey, Signature)>>, Box<dyn error::Error>> {
-    matches.try_contains_id(name)?;
-    Ok(pubkeys_sigs_of(matches, name))
+    Ok(matches.try_get_many::<String>(name)?.map(|values| {
+        values
+            .map(|pubkey_signer_string| {
+                let mut signer = pubkey_signer_string.split('=');
```

Comment:
prefer `splitn()` to avoid succeeding on invalid inputs

Reply:
Yep, valid point. I ended up using `split_once`, which also returns an option.